src/Renci.SshNet/Renci.SshNet.csproj

```diff
@@ -5,20 +5,24 @@
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <AssemblyName>Renci.SshNet</AssemblyName>
     <AssemblyOriginatorKeyFile>../Renci.SshNet.snk</AssemblyOriginatorKeyFile>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>latest</LangVersion>
```

Comment:
Please rebase. This and other properties have been removed, and are now set [here](https://github.com/sshnet/SSH.NET/blob/develop/Directory.Build.props).